src/app/pagamentoplanos/component/pagamentoplanos.component.ts

```diff
@@ -1,15 +1,175 @@
 import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { isInteger } from '@ng-bootstrap/ng-bootstrap/util/util';
+import Swal from 'sweetalert2';
+import { ExternalService } from '../service/external.service';
+import { PagamentoplanosService } from '../service/pagamentoplanos.service';
 
 @Component({
   selector: 'app-pagamentoplanos',
   templateUrl: './pagamentoplanos.component.html',
   styleUrls: ['./pagamentoplanos.component.css']
 })
 export class PagamentoplanosComponent implements OnInit {
+  cardForm: any;
 
-  constructor() { }
+  Toast = Swal.mixin({
+    toast: true,
+    position: 'top-end',
+    showConfirmButton: false,
+    timer: 3000,
+    timerProgressBar: true,
+    didOpen: (toast) => {
+      toast.addEventListener('mouseenter', Swal.stopTimer);
+      toast.addEventListener('mouseleave', Swal.resumeTimer);
+    },
+  });
+
+  constructor(
+    private scriptService: ExternalService,
+    private pagamentoSerice: PagamentoplanosService,
+    private router: Router
+  ) { }
 
   ngOnInit(): void {
+    this.pagamentoSerice.ngOnInit();
+    this.scriptService.load('MercadoPago').then(data => {
+  }).catch(error => console.log(error));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
  }).catch((error) => console.log(error));
```